hotline/urls.py

```diff
@@ -29,7 +29,7 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^$', home, name="home"),
     url(r'^adminpanel/?$', species.admin_panel, name='admin-panel'),
-    url(r'^species/list/?$', permissions.is_active(species.SpeciesList.as_view()), name='species-list'),
+    url(r'^species/list/?$', permissions.is_active(species.list_), name='species-list'),
```

Comment:
Since you turned the species list view into a function based view (from a class based view), you should move the permission decorator from this URL route, and onto the view function itself (just for consistency with all the other FBVs).